site/java/project-info.html

```diff
@@ -79,7 +79,7 @@ <h3><a name="Overview"></a>Overview</h3>
 <td align="left">This document lists the dependencies that are defined through dependencyManagement.</td></tr>
 <tr class="a">
 <td align="left"><a href="distribution-management.html">Distribution Management</a></td>
-<td align="left">This document provides informations on the distribution management of this project.</td></tr>
+<td align="left">This document provides information on the distribution management of this project.</td></tr>
```

Comment:
This will be overwritten next time the site is built (which will probably be immediately after this PR is merged), but I guess it doesn't hurt.